### ISSUE TITLE
Update audits/views for Service Locations changes

### ISSUE THREAD

**Issue by omahane** [department-of-veterans-affairs/va.gov-cms#16300]
## User Story or Problem Statement

This view: https://va-gov-cms.ddev.site/admin/content/facilities/vamc-facility-health-services
will need to change when service location changes are rolled out.

Views to change
- [x] /admin/content/audit/service-locations-detailed
- [x] /admin/content/facilities/vamc-facility-health-services

## Deleted fields
 field_hservice_appt_intro_select
 field_hservice_appt_leadin
 field_online_scheduling_availabl
 field_walk_ins_accepted
 field_phone_numbers_paragraph

## Screenshot
![Screenshot 2023-12-04 at 11 17 21 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/766573/0cee2abd-06af-4a60-9801-46a0f75c2ae2)

## Acceptance Criteria
- [x] Remove deleted fields from views
  - [x] field_hservice_appt_intro_select
  - [x] field_hservice_appt_leadin
  - [x] field_online_scheduling_availabl
  - [x] field_walk_ins_accepted
  - [x] field_phone_numbers_paragraph
- [x] Create follow up ticket for service-locations-detailed to include more Service location fields
- [ ] Requires Content Model Documentation
- [ ] Requires design review
- [ ] Requires KB article update

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`

**Comment by omahane** [comment]
@xiongjaneg ACs will need flushing out. Plus, it needs to be grouped with Service Locations work.

**Comment by xiongjaneg** [comment]
Will this view break if we roll out the integration branch?

No, but it will break when we do the removals so we want to time this before that.

**Comment by Becapa** [comment]
I do have a branch with the changes for this at https://github.com/department-of-veterans-affairs/va.gov-cms/tree/VACMS-16300-update-audits-for-service-location-changes. This change was made against the `integration-service-location` branch but will ultimately need to be merged into `main` once the integration branch has been merged and verified. The reason for this is that the current views will need to be in place to make sure that the migration is successful. Time got away from me yesterday afternoon and I wasn't able to make the PR. But since it won't be able to get merged in for a while anyways, I think that is okay and I'll get to it first thing next week.
cc. @xiongjaneg  @jilladams

**Comment by jilladams** [comment]
Updating assignment since Michael's moved on. When it's time to integrate, this ticket has its own branch to merge, after integration branch merges.   https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17479

**Comment by jilladams** [comment]
Service Locations have launched. We will check with VHA DM re: reasonable timeframe to expect Editors to verify their data. 

Once we get the all clear there, we can proceed with this PR. This, thusly, is actually still blocked. Marking as such.

**Comment by davidmpickett** [comment]
@jilladams @Agile6MSkinner 
I think we should revisit the assumption that this is blocked:
- this audit view is not visible to editors
- we're not going to ask all editors to review all Service Locations (we're confident in our migration)
- removing the fields from the audit views doesn't remove the data from Drupal (no risk of lost data)

**Comment by jilladams** [comment]
In sprint planning, we determined that #18145 will be limited to comms around 4 editors / sites with phone numbers that need to be revisited. Audit views aren't affected, this is unblocked. Removing blocked label.

**Comment by jilladams** [comment]
That said: we didn't discuss effect on capacity in sprint. The PR work is done, just needs refreshed / passing tests, and re-reviewed for merge:  https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17479 - I updated branch / CI will rerun. Should be low impact if CI passes.

**Comment by jilladams** [comment]
Verified in prod
